ranger/container/history.py

```diff
@@ -89,6 +91,11 @@ def rebase(self, other_history):
     def __len__(self):
         return len(self.history)
 
+    def _update_maxlen_(self, maxlen=None):
+        self.maxlen = maxlen if maxlen else self.settings.max_console_history_size
```

Comment:
You can't default to a specific setting because the History class needs to work for several.